test/resources/microshift-process.resource

```diff
@@ -55,11 +55,7 @@ MicroShift Process ID Changed
     [Documentation]    Compare the specified and current MicroShift process ID
     [Arguments]    ${old_pid}
     ${cur_pid}=    MicroShift Process ID
-    IF    "${cur_pid}"=="${old_pid}"
-        RETURN    False
-    ELSE
-        RETURN    True
-    END
+    Should Not Be Equal    ${cur_pid}    ${old_pid}
```

Comment:
This statement will fail the test if not true. The original idea here is to return the current status and retry the function in a loop. 
Why do we need this change?

Reply:
The function is enclosed in `Wait Until Keyword Succeeds` builtin. According to the [docs](https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Wait%20Until%20Keyword%20Succeeds) this will retry on almost any failure.
The original function was only returning True/False strings, which are not checked by the builtin so it succeeded on the first try, everytime.

Reply:
Ah, the function did not work as intended. Can you please rebase on top of the latest code in `main`? 
We have added another similar function for Etcd checks.

Reply:
Fixed in etcd process too. Thanks!